components/Payment/Form.js

```diff
@@ -195,10 +195,10 @@ class PaymentForm extends Component {
           amount: total,
           usage: 'reusable',
           card: {
-            number: values.cardNumber,
-            cvc: values.cardCVC,
-            exp_month: values.cardMonth,
-            exp_year: values.cardYear
+            number: values.cardNumber.trim(),
```

Comment:
```suggestion
            number: values.cardNumber && values.cardNumber.trim(),
```
callers of `createStripeSource` generally check that `values.cardNumber` is not empty but let's play it safe. Getting a stripes api response error is better than an exception.